portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,11 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// Cycles through 3 different pictures of myself.
+let pictureNumber = 0;
+function cyclePictures() {
```

Comment:
Usually comments preceding functions take the form of "doc strings." The format is a multi-line comment with two asterisks directly before the function declaration.

```javascript
let picutreNumber = 0;

/** Cycles through 3 different pictures of myself. */
function cyclePictures() {
```

These doc strings can usually be picked up automatically by code editors and documentation generation utilities to provide additional information about the function or method.

No action necessary.